ethereum/testdata/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
You just created the `testdata` directory for later, or why?

Reply:
Earlier I was using some fixed data for tests which were put in `testdata`. Then i rewrote the tests to not have any fixed data for tests. Hence `testdata` is not  required. I have removed it.